src/test/polymer/polymer-element-scanner_test.ts

```diff
@@ -116,6 +116,17 @@ suite('PolymerElementScanner', () => {
         ['all', 'Object']
       ]);
 
+      assert.deepEqual(features[0].attributes.map(p => [p.name, p.changeEvent]), [
+        ['a', undefined],
+        ['b', undefined],
+        ['c', undefined],
+        ['d', undefined],
+        ['e', 'e-changed'],
+        ['f', undefined],
+        ['g', undefined],
```

Comment:
Oh oops that's because I did not rebase this branch onto master. Since my other PR got merged which fixed short-hand property notation